action.yml

```diff
@@ -71,6 +71,10 @@ inputs:
     description: 'Upload binaries to specified release page that indicated by release name.'
     required: false
     default: ''
+  release_repo:
```

Comment:
Would you please add this feature in README.md as well? If you'll use it in production, I'll merge it and release a new version then. Thanks!

Reply:
Thanx for the heads-up @wangyoucao577, are you able to test-run this before building a new release?

I added it to the documentation but I prefer leaving it out of the YAML examples - most organizations will probably release on the same repository, those that need to release on a different repo will probably dig and read the doc instinctively

Reply:
I can't test it. So it'll be great if you can help to test it before I release the version!

Reply:
I added a real-life example, updated the PR's comment with links to the repositories. Thanks @wangyoucao577 for the support!

Reply:
I have merged this PR, could you please help to test it with `wangyoucao577/go-release-action@master`? If it works as expect, I'll release a new version for it. Thanks for your contribution!

Reply:
Works good:

- https://github.com/serafdev/go-release-action-test-private/releases/tag/v0.0.1-alpha.2
- https://github.com/serafdev/go-release-action-test-public/releases/tag/v0.0.1-alpha.2

Link to the pipeline: https://github.com/serafdev/go-release-action-test-private/actions/runs/4397561683

Thanks bud for your time! 🥇